strawberry/flask/tests/test_graphqlview.py

```diff
@@ -0,0 +1,33 @@
+import json
+
+import pytest
+
+from flask import url_for
+
+from .app import init_app
+
+
+@pytest.fixture
+def app():
+    return init_app()
+
+
+def test_graphql_query(client):
+    query = {"query": "query {\n  user {\n    name\n    age\n  }\n}"}
```

Comment:
let's make this into a multiline string, should be easier to read ;)

Reply:
Will do